src/CoaddPsf.cc

```diff
@@ -221,6 +221,30 @@ void addToImage(
 }
 
 
+afw::geom::Box2I CoaddPsf::doComputeBBox(
+    afw::geom::Point2D const & ccdXY,
+    afw::image::Color const & color
+) const {
+    afw::table::ExposureCatalog subcat = _catalog.subsetContaining(ccdXY, *_coaddWcs, true);
+    if (subcat.empty()) {
+        throw LSST_EXCEPT(
+            pex::exceptions::InvalidParameterError,
+            (boost::format("Cannot compute BBox at point %s; no input images at that point.")
+             % ccdXY).str());
+    }
+
+    afw::geom::Box2I overallBBox;
+    for (afw::table::ExposureCatalog::const_iterator i = subcat.begin(); i != subcat.end(); ++i) {
```

Comment:
Oh, I'm an idiot.  Of course; I was thinking that `i` would be an iterator type, not the record type.  However, I think the appropriate invocation is then `auto const & record`; we want the element type to resolve to an lvalue const reference `ExposureRecord const &`, not an rvalue reference `ExposureRecord &&`.